eng/pipelines/build.yml

```diff
@@ -178,68 +169,81 @@ stages:
         parameters:
           name: Check_Code_Generation_N_Z
           filter: "^[n-z]"
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
-    - template: pipelines/stages/net-release-to-feed.yml@azure-sdk-build-tools
-      parameters:
-        ShouldTag: false
-        ShouldSign: false
-        ShouldPublishSymbols: false
-    - stage: 'Update_azure_sdk_for_net'
-      dependsOn:
-        - Build_and_Test
-        - Release
-      variables:
-        AutorestCSharpVersion: $[stageDependencies.Build_and_Test.Build.outputs['Package.AutorestCSharpVersion']]
-        TypeSpecEmitterVersion: $[stageDependencies.Build_and_Test.Build.outputs['Publish.TypeSpecEmitterVersion']]
-        BranchName: "auto-update-autorest-$(AutorestCSharpVersion)"
-      jobs:
-        - template: update-generator-versions.yml
-          parameters:
-            Name: Update_Generator_Version
-            AutorestCSharpVersion: $(AutorestCSharpVersion)
-            BranchName: $(BranchName)
-        - template: generate-sdk-job-matrix-files.yml
-          parameters:
-            Name: Generate_Job_Matrix
-            DependsOn: Update_Generator_Version
-            JobCount: 7
-            BranchName: $(BranchName)
-        - template: update-azure-sdk-for-net-codes.yml
-          parameters:
-            Name: Update_Code
-            DependsOn: Generate_Job_Matrix
-            Matrix: $[dependencies.Generate_Job_Matrix.outputs['generate_job_matrix.matrix']]
-            AutorestCSharpVersion: $(AutorestCSharpVersion)
-            BranchName: $(BranchName)
-            ContinueOnError: true
-        - job: Create_PR
-          dependsOn: Update_Code
-          variables:
-            UpdateCodeResult: $[dependencies.Update_Code.result]
-          steps:
-            - checkout: self
-            - checkout: azure-sdk-tools
-            - pwsh: |
-                if ('$(UpdateCodeResult)' -eq 'Succeeded') {
-                  $prTitle = 'Update AutoRest C# version to $(AutorestCSharpVersion)'
-                } else {
-                  $prTitle = 'Failed Update AutoRest C# version to $(AutorestCSharpVersion)'
-                }
+      - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+        stage: Release
+        pool:
+          name: $(LINUXPOOL)
+          image: $(LINUXVMIMAGE)
+          os: linux
+        jobs:
+          job:
+          - task: NuGetCommand@2
```

Comment:
This will need replacing with the 1es templateContext or nuget task.

Reply:
Change do 1es nuget task